flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg

```diff
@@ -86,53 +86,49 @@ point.8 = 2, -75.20,   0,  32.3,   100,  0,       0,   0,      0,     0,     0,
 fuel_type = 2 ; 1 = OCTANE 100, 2 = JET_A, 3 = OCTANE 80, 4 = AUTO GAS, 5 = JET B
 
 [FUEL_SYSTEM]
+Version = Latest
```

Comment:
Best to fix this to the latest version number so it won't get broken by a future update to the sim.
[From Sylvain](https://devsupport.flightsimulator.com/questions/11585/fuel-system-bug-junctions-cannot-have-all-inputs-c.html?childToView=13404#answer-13404):
> Also switch your version from Latest to 4 for SU11. Latest is a very risky setting as it will change between updates.

Reply:
Nit so sure about that. They talk out of both sides of their mouth. They say that, but notice that the default airplanes use "Version = Latest."